build.sh

```diff
@@ -15,8 +15,9 @@ function usage()
   echo "  -h|--help               print the help info"
   echo "  -d|--debug              CMAKE_BUILD_TYPE=Debug (default is RelWithDebInfo)"
   echo "  -l|--language python(default) "
-  echo "                          build native library for python"
-  echo "                java      build native library for java"
+  echo "                                  build native library for python"
+  echo "                java              build native library for java"
+  echo "                python,java       build native library for java & python"
```

Comment:
yes, we have use cases where only need java build. The reason why I prefer to make this option a list is because it's easy to extend. If we support more languages in the future, users can specify any combination of these languages in this option.

Reply:
But here, we can make the doc more clear by saying this option should be a comma-separated list of languages to build.
```shell
echo "  -l|--language languages  a list of languages to build native libraries, separated by commas."
echo "                           Supported languages include python and java. If nothing is specified,"
echo "                           only python library will be built."
```